### PR TITLE
feat(api): support listing pipelines triggered by pipeline schedules

### PR DESCRIPTION
**Pull request by svermeulen**

### CODE REVIEW

File: gitlab/v4/objects/pipelines.py
Comment (nejch) [inline gitlab/v4/objects/pipelines.py:194]: We can use the singular for the managers here I think. Also `id` should be enough, the encoding is done elsewhere:

```suggestion
class ProjectPipelineSchedulePipelineManager(ListMixin, RESTManager):
    _path = "/projects/{project_id}/pipeline_schedules/{pipeline_schedule_id}/pipelines"
    _obj_cls = ProjectPipeline
    _from_parent_attrs = {"project_id": "project_id", "pipeline_schedule_id": "id"}


class ProjectPipelineSchedule(SaveMixin, ObjectDeleteMixin, RESTObject):
    variables: ProjectPipelineScheduleVariableManager
    pipelines: ProjectPipelineSchedulePipelineManager
```

**Review by nejch — CHANGES_REQUESTED** [review]
Thanks a lot for the contribution @svermeulen! The commit message will need a little `--amend` as we generate changelogs from these - see https://github.com/python-gitlab/python-gitlab/blob/main/CONTRIBUTING.rst#commit-message-guidelines.

I added a small note. 

It'd be great if you could also add a little example here:  
https://github.com/python-gitlab/python-gitlab/blob/main/docs/gl_objects/pipelines_and_jobs.rst#examples-2

And if you're comfortable writing pytest tests, I'd also add a unit test for this here:  
https://github.com/python-gitlab/python-gitlab/blob/main/tests/unit/objects/test_pipeline_schedules.py

If not, just let me know and we'll add the tests as a follow-up so we can get this merged for the release on the 28th. Thanks again!

**Comment by nejch** [comment]
@svermeulen we'll have a release scheduled in 2 days so we can potentially merge this in without the tests/docs if you look at the  notes above, and I'll do the rest as follow-up :)

**Comment by nejch** [comment]
Hi @svermeulen! Let us know if you'd like to finish this PR, or if we should push to your branch to get this merged (either way you'll be credited as the commit author). Thanks again for the contribution!

**Comment by svermeulen** [comment]
Hi @nejch 
Sorry, I haven't been able to find the time since sending the original PR here and not sure how soon that will change, so if you or someone else wants to take it over please feel free!

File: gitlab/v4/objects/pipelines.py
Comment (nejch) [reply gitlab/v4/objects/pipelines.py]: Done in 202bbdd0da5baf138d7e794451d49eccb623559f.

**Review by nejch — APPROVED** [review]
LGTM, did a little fix up and I've added tests in 2db5543313c424b67dcfff34707dba3a3a52b1ee so would need another pair of eyes for merge.

**Comment by nejch** [comment]
@lmilbaum we are a bit short on maintainers at the moment, if you'd like to give this one a look let me know and I'll merge once we have one more approval ;)

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**
